docs/resources/Soundboard.md

```diff
@@ -0,0 +1,90 @@
+# Soundboard Resource
+
+Users can play soundboard sounds in voice channels, triggering a Voice Channel Effect Send Gateway event for users connected to the voice channel. Bots cannot play soundboard sounds. Soundboard use is controlled by the `USE_SOUNDBOARD` and `USE_EXTERNAL_SOUNDS` [permissions](#DOCS_TOPICS_PERMISSIONS).
```

Comment:
```suggestion
Users can play soundboard sounds in voice channels, triggering a [Voice Channel Effect Send](#DOCS_TOPICS_GATEWAY_EVENTS/voice-channel-effect-send) Gateway event for users connected to the voice channel. Bots cannot play soundboard sounds. Soundboard use is controlled by the `USE_SOUNDBOARD` and `USE_EXTERNAL_SOUNDS` [permissions](#DOCS_TOPICS_PERMISSIONS).
```

Reply:
this will make the ci fail until #6025 gets merged

Reply:
This (finally) shouldn't be an issue anymore. And the voice channel effect send GW event could get a link back to soundboard sounds as well now.